crowbar_framework/app/views/installer/upgrades_nav.html.haml

```diff
@@ -0,0 +1,49 @@
+%ul.nav.nav-stacked.wizard-nav.col-lg-3
```

Comment:
can we drop this file? as we have `crowbar_framework/app/views/installer/upgrades/_nav.html.haml` which looks almost the same